src/app/services/tvdb.service.ts

```diff
@@ -10,11 +10,21 @@ const API_BASE_URL = 'http://api.tvmaze.com';
 interface TvDbResponseItem {
   show: {
     name: string;
+    summary: string;
+    image: {
+      medium: string;
+    }
   };
 }
 
+function getPlaceholderUrl(title: string) {
+  return `https://via.placeholder.com/210x295?text=${encodeURI(title)}`;
+}
+
 const itemToShowData = (item: TvDbResponseItem): TvShowData => ({
   title: item.show.name,
+  summary: item.show.summary,
+  posterUrl: item.show.image ? item.show.image.medium : getPlaceholderUrl(item.show.name),
```

Comment:
If you wanted you could use `ramda`'s `ifElse` function for this.

Reply:
Will do.  I want to use Ramda wherever I can.